src/compiler/diagnosticMessages.json

```diff
@@ -2293,7 +2293,7 @@
         "category": "Error",
         "code": 2575
     },
-    "Property '{0}' is a static member of type '{1}'.": {
+    "Property '{0}' does not exist on type '{1}'. Did you mean to access the static member '{1}{2}' instead?": {

```

Comment:
Unclear if it should be `{1}{2}` - localization might get the hint from the surrounding `'`, but maybe just make it a `{3}` and use a 3rd argument.